rollouts/controllers/remoterootsync_controller.go

```diff
@@ -24,6 +24,7 @@ import (
 
 	apierrors "k8s.io/apimachinery/pkg/api/errors"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
```

Comment:
nit: In kubernetes code bases, convention is to use `metav1` alias for this package. It's right above this line.

Reply:
Thanks! Fixed with latest.